invokeai/backend/model_manager/load/model_cache/model_cache_base.py

```diff
@@ -115,6 +115,17 @@ def move_model_to_device(self, cache_entry: CacheRecord[AnyModel], target_device
         """Move model into the indicated device."""
         pass
 
+    @abstractmethod
+    def get_execution_device(self) -> torch.device:
+        """
+        Return a GPU device in a multi-GPU environment.
+
+        Device should be reserved with reserve_gpu_device()
+        before calling this. If not, it will return the
+        default device.
+        """
+        pass
+
```

Comment:
This is never called and the concrete method raises a `NotImplementedError` if it is. Can we move this change to the multi-GPU PR please?

Reply:
Done.